xml/hwbp_registers.xml

```diff
@@ -987,10 +995,17 @@
             If the H extension is not supported, the only legal values are 0 and 4.
         </field>
 
-        <field name="0" bits="22:18" access="R" reset="0" />
+        <field name="0" bits="22:20" access="R" reset="0" />
+        
+        <field name="sbytemask" bits="19:18" access="WARL" reset="0">
+            0=compare, 1=don't compare byte, i.e. byte is masked out
+            bit 18 = lower byte compare of svalue (bits [7:0])
+            bit 19 = upper byte compare of svalue (bits [15:8])
+        </field>
 
         <field name="svalue" bits="17:2" access="WARL" reset="0">
-            Data used together with \FcsrTextraThirtytwoSselect.
+            Data used together with \FcsrTextraThirtytwoSselect to compare with the scontext CSR
+            which is ANDed with all other trigger comparisons.
```

Comment:
I would suggest leaving this text unchanged since svalue can compare to either scontext or ASID depending on sselect.  The text already makes it clear (I think) that all comparisons associated with a trigger must all match (ANDed) for the trigger to match.